documentation/components/tutorial-flow-shortcut.asciidoc

```diff
@@ -108,26 +117,22 @@ public class Scope extends Div {
 }
 ----
 
-The shortcut in this example is tied to the parent component of the `Input`
-components, `Scope`. If the user is typing something in one of the inputs and
-presses _Escape_, both input fields will be cleared and focus returned to the
-first field.
+* The shortcut is tied to the parent component (`Scope`) of the input components. 
```

Comment:
The whole example is quite confusing and don't work. I'll return to this later. Probably we'll modify the example code as well.

Reply:
"input components" isn't necessary very explicit and also it is not true that the shortcut is tied to those. As long as focus is inside the "scope", click shortcuts are executed.

Reply:
This is broken, but there is a ticket https://github.com/vaadin/flow/issues/5250#issuecomment-472734118 about it. I'll just want to get this stuff further.